programs/clearing_house/src/math/amm.rs

```diff
@@ -325,15 +351,27 @@ pub fn calculate_new_oracle_price_twap(
     amm: &AMM,
     now: i64,
     oracle_price: i128,
+    period: i64,
```

Comment:
this might be be nicer/safer as an enum:
```
pub enum TwapPeriod {
    FundingPeriod,
    FiveMin,
}

pub fn calculate_new_oracle_price_twap(
    amm: &AMM,
    now: i64,
    oracle_price: i128,
    twap_period: TwapPeriod,
) -> ClearingHouseResult<i128> {
    match twap_period {
        TwapPeriod::FundingPeriod => (amm.last_mark_price_twap, amm.last_oracle_price_twap),
        TwapPeriod::FiveMin => (
            amm.last_mark_price_twap_5min,
            amm.last_oracle_price_twap_5min,
        ),
    }